client/valor/schemas/symbolic/static_collections.py

```diff
@@ -0,0 +1,367 @@
+import typing
+from typing import Any, Optional
+
+from valor.enums import TaskType
+from valor.schemas.symbolic.annotations import (
+    BoundingBox,
+    BoundingPolygon,
+    Embedding,
+    Raster,
+    Score,
+    TaskTypeEnum,
+)
+from valor.schemas.symbolic.atomics import Equatable, String, Symbol, Variable
+from valor.schemas.symbolic.structures import (
+    Dictionary,
+    List,
+    _get_atomic_type_by_name,
+)
+
+
+def _get_schema_type_by_name(name: str):
+    name = name.lower()
+    if name == "score":
+        return Score
+    elif name == "boundingbox":
+        return BoundingBox
+    elif name == "boundingpolygon":
+        return BoundingPolygon
+    elif name == "raster":
+        return Raster
+    elif name == "embedding":
+        return Embedding
+    elif name == "dictionary":
+        return Dictionary
+    elif name == "label":
+        return Label
+    elif "list[label]" in name:
+        return List[Label]
+    elif name == "annotation":
+        return Annotation
+    elif "list[annotation]" in name:
+        return List[Annotation]
+    elif name == "datum":
+        return Datum
+    else:
+        return _get_atomic_type_by_name(name)
+
+
+class StaticCollection(Equatable):
```

Comment:
nit: this name doesn't seem intuitive to me, but I can't think of a better alternative

Reply:
My reasoning is that it a collection of static class members